tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+      default: "rhtap-releng-tenant/rpa" # This needs to change later
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: releasePlanAdmission
+      type: string
+      description: The relative path in the workspace to the stored releasePlanAdmission json
+  steps:
+    - name: create-exodus-conf-file
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
+      env:
+        - name: "RELEASE_PLAN_ADMISSION"
```

Comment:
This variable is created, but then the script doesn't use it and uses the `$(params.*)` reference anyway. So either this can be removed, or the env var should be used in the script.

But I think the RPA is probably not needed anyway - see my other comment.

Reply:
Stopped using RELEASE_PLAN_ADMISSION directly, updated to use dataPath instead.

Reply:
Cool.